src/dep/triemap/triemap.c

```diff
@@ -266,6 +266,57 @@ void *TrieMapNode_Find(TrieMapNode *n, char *str, tm_len_t len) {
   return TRIEMAP_NOTFOUND;
 }
 
+int TrieMapNode_FindPrefixes(TrieMapNode *node, const char *str, tm_len_t len, arrayof(void*) *results) {
```

Comment:
Why not just run with an iterator over the subtree?

Reply:
It's for finding all prefixes of a given key that have nodes in the trie (each node may have several matching rules). An iterator can get the suffixes of the key. But maybe it's really better to implement it as another kind of iterator.